Workbooks/ActivityLogsInsights/AtScale/ActivityLogsInsights.workbook

```diff
@@ -1,40 +1,89 @@
 {
   "version": "Notebook/1.0",
   "items": [
-    {
-      "type": 1,
-      "content": {
-        "json": "**NOTE**\n* It is required to enable Log Analytics in Diagnotic Settings for this workbook to work."
-      },
-      "name": "text - 0"
-    },
     {
       "type": 9,
       "content": {
         "version": "KqlParameterItem/1.0",
         "parameters": [
           {
-            "id": "5de89c5b-d42e-4109-ad41-b08cae2e233e",
+            "id": "0e85e0e4-a7e8-4ea8-b291-e444c317843a",
             "version": "KqlParameterItem/1.0",
-            "name": "CurrentResource",
-            "type": 5,
+            "name": "ResourceTypes",
+            "label": "Resource types",
+            "type": 7,
             "isRequired": true,
+            "multiSelect": true,
+            "quote": "'",
+            "delimiter": ",",
+            "value": [
+              "value::all"
+            ],
+            "isHiddenWhenLocked": true,
             "typeSettings": {
               "additionalResourceOptions": [
-                "value::1"
+                "value::all"
               ],
-              "componentIdOnly": true
+              "includeAll": true,
+              "showDefault": false
+            }
+          },
+          {
+            "id": "1f74ed9a-e3ed-498d-bd5b-f68f3836a117",
+            "version": "KqlParameterItem/1.0",
+            "name": "Subscription",
+            "label": "Subscriptions",
+            "type": 6,
+            "description": "All subscriptions with Storage accounts",
```

Comment:
this description was probably copied from another template?

Reply:
done